src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/Util/WriteUtils.vb

```diff
@@ -617,7 +617,7 @@ Public MustInherit Class WriteUtils
             text = XmlEscape(text)
         Else
             ' Ensure the text does not require escaping.
-            Dim filtered = text.Replace("<cref ", "").Replace("/>", "").Replace("&amp;", "").Replace("&lt;", "").Replace("&gt;", "")
+            Dim filtered = text.Replace("<see ", "").Replace("/>", "").Replace("&amp;", "").Replace("&lt;", "").Replace("&gt;", "")
```

Comment:
shouldn't this be `"<see cref=", ""`?

tbh, i'm not really sure what is going on here...

Reply:
The logic in `XmlEscape` on next line only looks for `&`, `<` and `>`. It doesn't look for `=` or `"`. 
So I think this is correct.

Reply:
It looks like the result of this transformation is only used for a debug assert. e.g., if user says "don't escape the text", then verify in an ad-hoc manner that the text didn't require escaping.

Basically, since "cref" itself doesn't need to be escaped, we don't need to check for it and replace here.